src/manifest.c

```diff
@@ -537,15 +538,18 @@ static int retrieve_manifests(int current, int version, char *component, struct
 		goto out;
 	}
 
-	string_or_die(&tar, TAR_COMMAND " -C %s/%i -xf %s/%i/Manifest.%s.tar 2> /dev/null",
-		      state_dir, version, state_dir, version, component);
-
-	/* this is is historically a point of odd errors */
```

Comment:
As much as I *hope* we do not see errors here anymore, can you restore this comment, for reference?